RealEstate/apps/core/templates/core/base.html

```diff
@@ -21,67 +21,77 @@
   </head>
 
   <body>
-    <nav class="navbar navbar-default">
-      <div class="container-fluid">
-        <!-- Brand and toggle get grouped for better mobile display -->
-        <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
-                  data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class="navbar-brand" href="{% url 'home' %}">
-            <span class="glyphicon glyphicon-home" aria-hidden="true"></span> Real Estate App
-          </a>
-        </div>
+    {% if user.is_active %}
```

Comment:
Can you change the is_active checks to is_authenticated?  is_active is a check for whether or not the account is active, ie have they deactivated it or not.  is_authenticated checks if they are currently logged in.